en/docs/install-and-setup/setup/distributed-deployment/deploying-wso2-api-m-in-a-distributed-setup.md

```diff
@@ -1,466 +1,227 @@
-    type = "mysql"
-    hostname = "db.wso2.com"
-    name = "apim_db"
-    port = "3306"
-    username = "root"
-    password = "root"
-    
-    [database.shared_db]
-    type = "mysql"
-    hostname = "db.wso2.com"
-    name = "shared_db"
-    port = "3306"
-    username = "root"
-    password = "root"
-    
-    [keystore.tls]
-    file_name =  "wso2carbon.jks"
-    type =  "JKS"
-    password =  "wso2carbon"
-    alias =  "wso2carbon"
-    key_password =  "wso2carbon"
-    
-    [truststore]
-    file_name = "client-truststore.jks"
-    type = "JKS"
-    password = "wso2carbon"
-    
-    [apim.throttling]
-    throttle_decision_endpoints = ["tcp://tm.wso2.com:5675"]
+## Step 6 - Configure and start the profiles
 
-    [[apim.throttling.url_group]]
-    traffic_manager_urls=["tcp://tm.wso2.com:9614"]
-    traffic_manager_auth_urls=["ssl://tm.wso2.com:9714"]
+Let's configure the API-M nodes in the deployment.
 
-    [[event_listener]]
-    id = "token_revocation"
-    type = "org.wso2.carbon.identity.core.handler.AbstractIdentityHandler"
-    name = "org.wso2.is.notification.ApimOauthEventInterceptor"
-    order = 1
-
-    [event_listener.properties]
-    notification_endpoint = "https://tm.wso2.com:9446/internal/data/v1/notify"
-    username = "${admin.username}"
-    password = "${admin.password}"
-    'header.X-WSO2-KEY-MANAGER' = "default"
-    
-    [apim.cache_invalidation]
-    enabled = true
-    domain = "km-domain"
-    ```
+### Configure the Gateway nodes
 
-### Step 6.2 - Configure and start the Traffic Manager
+Configure the Gateway nodes in the deployment to communicate with the Control Plane nodes.
 
-This section involves setting up the Traffic Manager node(s) and enabling it to work with the other components in a distributed deployment.
+<a href="{{base_path}}/assets/img/setup-and-install/gateway-connections.png"><img src="{{base_path}}/assets/img/setup-and-install/gateway-connections.png" width=600></a>
 
-<a href="{{base_path}}/assets/img/setup-and-install/traffic-manager-connections.png"><img src="{{base_path}}/assets/img/setup-and-install/traffic-manager-connections.png" width="500"></a>
+Follow the steps given below.
+                                                         
+1.  Open the `<API-M_HOME>/repository/conf/deployment.toml` file of the Gateway node.
 
+2.  Add the following configurations to the `deployment.toml` file.
 
-1.  Open the `<API-M_HOME>/repository/conf/deployment.toml` file in the Traffic Manager node and change as following to point to Key Manager nodes.
+    -   Connecting the Gateway to the Key Manager component in the Control Plane:
 
-    ``` toml tab="Key Manager with HA"
-    [apim.key_manager]
-    service_url = "https://[Key-Manager-LB-host]/services/"
-    username = "$ref{super_admin.username}"
-    password = "$ref{super_admin.password}"
-    ```
+        ``` toml tab="Key Managers with HA"
+        [apim.key_manager]
+        service_url = "https://[Key-Manager-LB-host]/services/"
```

Comment:
It is actually the Control Plane but we are connecting to the Key manager component. But it may be clearer if we say use Control Plane. Will change that.

Reply:
fixed